server/database/database.js

```diff
@@ -11,6 +11,7 @@ const db = mongoose.connection;
 //create your schemas here
 
 const UserSchema = mongoose.Schema({
+  name: String,
```

Comment:
This will need something for `session` but I can add that if it's alright with you.

Reply:
That's ok with me.